backend/src/main/java/com/woowacourse/levellog/prequestion/exception/PreQuestionNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package com.woowacourse.levellog.prequestion.exception;
+
+import com.woowacourse.levellog.common.exception.LevellogException;
+import org.springframework.http.HttpStatus;
+
+public class PreQuestionNotFoundException extends LevellogException {
+
+    public PreQuestionNotFoundException(final String message, final String clientMessage) {
+        super(message, clientMessage, HttpStatus.NOT_FOUND);
+    }
```

Comment:
```suggestion
    private static final String ERROR_MESSAGE = "사전 질문이 존재하지 않습니다.";
    
    public PreQuestionNotFoundException(final String message) {
        super(message, ERROR_MESSAGE, HttpStatus.NOT_FOUND);
    }
```
이건 어떨까요? 클라이언트 입장에서는 사전 질문인지 작성한 사전 질문인지 알 필요가 없어 보여서요 😁

Reply:
수정했습니다